src/lib/shim/preload_libraries.c

```diff
@@ -652,4 +652,4 @@ struct tm* localtime(const time_t* timep) {
 struct tm* localtime_r(const time_t* timep, struct tm* result) {
     // Return time relative to UTC rather than the locale where shadow is being run.
     return gmtime_r(timep, result);
-}
+}
```

Comment:
I don't think we want to remove the new line from this file, and it's the only change in the file. (And the same for `src/test/tor/minimal/CMakeLists.txt`).

Reply:
Done